src/check/parse/IR.zig

```diff
@@ -596,6 +608,11 @@ pub const NodeStore = struct {
                 node.data.lhs = mod.exposes.span.start;
                 node.data.rhs = mod.exposes.span.len;
             },
+            .malformed => |a| {
```

Comment:
Thank you!  I just said this morning that I need to do exactly this.  Looks great for the most part.

Having region will help with this quite a bit, will be adding that to node soon

Reply:
Yes, region will mean we can provide much nicer formatted error messages